extensions/cornerstone/src/utils/stackSync/toggleStackImageSync.ts

```diff
@@ -1,90 +1,80 @@
-import calculateViewportRegistrations from './calculateViewportRegistrations';
+const STACK_SYNC_NAME = 'stackImageSync';
```

Comment:
Handled by the CS3D tool library - that way there is exactly one spot to do the calculations.

Reply:
Also, I haven't removed calculateViewportRegistrations because it is needed for another use case which we don't yet support, allowing one to adjust hte registrations.